<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ndvi</class>
 <widget class="QDialog" name="Ndvi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NDVI - Normalized Difference Vegetation Index</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Information:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>84</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>RED Band:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NIR Band:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Output File Path:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="ndviRED_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ndviNIR_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ndviOut_txt"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>82</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ndvi_btn">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>84</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="ndviRED_btn">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ndviNIR_btn">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ndviOut_btn">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>ndvi_btn</zorder>
  <zorder>ndviRED_txt</zorder>
  <zorder>ndviNIR_txt</zorder>
  <zorder>ndviOut_txt</zorder>
  <zorder>ndviRED_btn</zorder>
  <zorder>ndviNIR_btn</zorder>
  <zorder>ndviOut_btn</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>ndviRED_btn</zorder>
  <zorder>label_4</zorder>
  <zorder>cancel_btn</zorder>
  <zorder>ndvi_btn</zorder>
  <zorder>cancel_btn</zorder>
  <zorder>cancel_btn</zorder>
  <zorder>ndvi_btn</zorder>
  <zorder>textBrowser</zorder>
  <zorder>label_5</zorder>
 </widget>
 <tabstops>
  <tabstop>ndviRED_btn</tabstop>
  <tabstop>ndviNIR_btn</tabstop>
  <tabstop>ndviOut_btn</tabstop>
  <tabstop>ndvi_btn</tabstop>
  <tabstop>ndviRED_txt</tabstop>
  <tabstop>ndviNIR_txt</tabstop>
  <tabstop>ndviOut_txt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>Ndvi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
